docker-hub/publish/insights-analytics.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Insights & analytics
+description: Provides an API analytics tool for your products.
+keywords: docker hub, hub, insights, analytics, api
+---
+
+The Insights & Analytics tool includes an API that provides analytics for your products. With this API, you now have self-serve access to metrics where you can download both raw data and summary data. You can view how many times your images have been pulled by tag or by digest, and get breakouts by geolocation, cloud provider, and tool.The API measures the number of 'docker pulls' that were triggered by users, and automated machines such as CICD systems. You can import this weekly or monthly data into your own systems or you can analyze it manually using spreadsheet reports. The data can then be exported as a downloadable GZipped CSV file. Review the Data Definitions section for more information.
+
+There are automated systems that frequently check for new versions of your images, and we have tried to distinguish between those 'version checks' and real user downloads by inspecting the order and timing of API calls from the same IP address. This will provide more insight into customer behavior. You can inspect the rules we have applied in the Action Classification Rule Set section of this document. We welcome [feedback](https://forms.gle/nb7beTUQz9wzXy1b6)(: target="_blank" rel="noopener" class="_") and questions on these rules.
+
+## Raw Data
+
+In your raw data CSV you will have access to the data points listed below. You can request raw data for a complete week (Monday to Monday) or for a complete month (available on the 1st day of the following month). Please note that each Action is represented as a single row.
+
+  * Type (industry)
+  * Host (cloud provider)
+  * Country (geolocation)
+  * Timestamp
+  * Namespace
+  * Repo
+  * Reference (digest will always be included, we provide tag when available)
+  * HTTP Method
+  * Action - can filter on the below
+  * Pull by tag
+  * Pull by digest
+  * Version check
+  * User-Agent Tool
+
+## Summary Data
+
+In your summary data CSV, you will have access to the data points listed below. You can request summary data for a complete week (Monday to Monday) or for a complete month (available on the 1st day of the following month).
+
+For For every namespace, repo, and reference (tag or digest):
+
+  * Unique IP address count
+  * Pulls by tag count
+  * Pulls by digest count
+  * Version Check count
+
+## Data Definitions
+
+| Data Point | Definition |
+|:-----|:--------|
+| Action | An Action represents the multiple request events associated with a docker pull. We have applied rules to these events so that the data is more meaningful in analyzing user behavior and intent. An Action can be filtered into three distinct categories: version check, pull by tag, and pull by digest. Each Action is represented as a single row in the raw data. |
```

Comment:
```suggestion
| Action | An Action represents the multiple request events associated with a `docker pull`. We have applied rules to these events so that the data is more meaningful in analyzing user behavior and intent. An Action can be filtered into three distinct categories: version check, pull by tag, and pull by digest. Each Action is represented as a single row in the raw data. |
```